RedditSharp/Multi/Multi.cs

```diff
@@ -0,0 +1,362 @@
+﻿using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Authentication;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace RedditSharp.Multi
+{
+    public class Multi
+    {
+        private Reddit Reddit { get; set; }
+        private IWebAgent WebAgent { get; set; }
+
+        #region Constant API URLs
+        private const string GetCurrentUserMultiUrl = "/api/multi/mine";
+        private const string GetPublicUserMultiUrl = "/api/multi/user/{0}";
+        private const string GetMultiPathUrl = "/api/multi/{0}";
+        private const string GetMultiDescriptionPathUrl = "/api/multi/{0}/description";
+        private const string GetMultiSubUrl = "/api/multi/{0}/r/{1}";
+        private const string PostMultiRenameUrl = "/api/multi/rename";
+        private const string PutSubMultiUrl = "/api/multi/{0}/r/{1}";
+        private const string CopyMultiUrl = "/api/multi/copy";
+
+        #endregion
+
+
+        public Multi(Reddit Reddit2, IWebAgent WebAgent2)
+        {
+            Reddit = Reddit2;
```

Comment:
use "this" syntax so that the variable names aren't weird, also variable names are traditionally whateverCaseThisIs while properties are CamelCase